src/Plugins/ApplicationLogs/ApplicationLogs.json

```diff
@@ -3,7 +3,8 @@
     "Path": "ApplicationLogs_{0}",
     "Network": 860833102,
     "MaxStackSize": 65535,
-    "Debug": false
+    "Debug": false,
+    "UnhandledExceptionPolicy": "StopPlugin"
```

Comment:
I think it's better in almost all of them `StopNode` by default, is the current behaviour

Reply:
nothoing was changed since the last review but the deledage invoke. Different plugin has different behavior on unhandled exception that are precisely set. Default behavior means nothing here.